RIE/RIE185bisand270bis.xml

```diff
@@ -159,35 +160,205 @@ schematypens="http://relaxng.org/ns/structure/1.0"
             <div type="textpart" subtype="face" xml:id="A">
 
                <div type="textpart" n="1" xml:id="RIE270bis" xml:lang="grc" corresp="LIT4851greekRoyal">
-                  <ab/>
-               </div>
-               <div type="textpart" n="5" xml:id="RIE185bisII3" xml:lang="gez" corresp="LIT4850pseudotrilingual">
-                  <head>D part 3</head>
-                  <ab/>
+                  <ab>
+                    <lb n="1"/> Ἀειζανᾶς βασιλεὺς Ἀξωμιτῶν καὶ Ὁμηριτῶν
+                    <lb n="2"/> καὶ τοῦ Ῥαειδᾶν καὶ Αἰθιόπων καὶ Σαβαειτῶν
+                    <lb n="3"/> καὶ τοῦ Σιλεῆ καὶ τοῦ Τιαμῶ καὶ Βουγαειτῶν <expan><abbr>κ</abbr><ex>αὶ</ex></expan>
+                    <lb n="4"/> τοῦ Κάσου, βασιλεὺς βασιλέων, υἱὸς θεοῦ ἀνική<lb n="5" break="no"/>
```

Comment:
mind the space. you may make a new line between '5"' and 'break', but where it is now, after the element, this does not exist :)

Reply:
some for others similar cases

Reply:
I'm sorry Pietro but I don't understand what I need to do. The word is devided across the lines 4 and 5, so I followed the Epidoc example, used to render 
"imper- 
ator": 
```<ab>
 <lb n="6"/>imper<lb n="7" break="no"/>ator
</ab>
```
Did I pick the wrong example? 
Thanks a lot

Reply:
from what I see here there a space after break="no"/>, which should not be there.